code/src/components/AddTodo.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+
+import { todos } from 'reducers/todos'
+
+
+const Form = styled.form`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+`
+
+const Textarea = styled.textarea`
```

Comment:
Suggestion:
Add resize: none | vertical; to prevent the user to resize the textarea horizontally

Reply:
cool! I didn't know this. I have added this :)